### PR TITLE
[feat: #837] Add mailto: link to the contributors list in the sidebar

### LINKED ISSUE #837: Add mailto: link to the contributors list in the sidebar
In the sidebar of the package detail view it would be nice if I could click on the contributor's name or avatar to send him/her an email.

Maybe with some metadata prefilled, like:

Subject: `${packageName}@${version} on ${verdaccioHost}"

### PR DESCRIPTION
**Pull request by dmstern**
**Description:**

Resolves #837 

**ToDo:** fix tests.

### CODE REVIEW

**Comment by juanpicado** [comment]
Two things:

* Rebase from `master`
* Run `yarn run test:unit -u` and I guess you only have to update the snapshots

**Comment by ayusharma** [comment]
@dmstern Please update commit message https://github.com/verdaccio/verdaccio/blob/master/CONTRIBUTING.md#git-commit-guidelines

File: src/webui/src/components/PackageSidebar/modules/Maintainers/MaintainerInfo/index.jsx [hunk 1]
```diff
@@ -3,17 +3,21 @@ import PropTypes from 'prop-types';
 
 import classes from './style.scss';
 
-export default function MaintainerInfo({title, name, avatar}) {
+export default function MaintainerInfo({title, name, avatar, email}) {
   let avatarDescription = `${title} ${name}'s avatar`;
+  let href= `mailto:${email}`;
```
[the remark this answers is not in the document]
Comment (juanpicado) [reply src/webui/src/components/PackageSidebar/modules/Maintainers/MaintainerInfo/index.jsx]: ping @dmstern

[the remark this answers is not in the document]
Comment (juanpicado) [reply src/webui/src/components/PackageSidebar/modules/Maintainers/MaintainerInfo/index.jsx]: ping @dmorgenstern-init

**Pull request closed without merging**